lib/v2/screens/profile_screens/set_currency/set_currency_screen.dart

```diff
@@ -83,6 +85,19 @@ class _SetCurrencyScreenState extends State<SetCurrencyScreen> {
     );
   }
 
+  String countryCodeToEmoji(Currency currency) {
```

Comment:
This method should live inside a mapper and called from the bloc.  No logic in the UI please.